jest-rs/README.md

```diff
@@ -0,0 +1,583 @@
+#Jest
+
+Jest is a Java HTTP Rest client for [ElasticSearch](http://www.elasticsearch.org).
+
+ElasticSearch is an Open Source (Apache 2), Distributed, RESTful, Search Engine built on top of Apache Lucene.
+
+ElasticSearch already has a Java API which is also used by ElasticSearch internally, [but Jest fills a gap, it is the missing client for ElasticSearch Http Rest interface](#comparison-to-native-api).
+
+>Read great [introduction](http://www.ibm.com/developerworks/java/library/j-javadev2-24/index.html?ca=drs-) to ElasticSearch and Jest from IBM Developer works.
+
+Installation
+------------
+
+### Stable
+
+Jest maven repository is hosted on [Sonatype](http://www.sonatype.org) which is then synced to [Maven Central](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22io.searchbox%22) with a slight delay.
+
+To get the latest *stable* version without waiting for Maven Central sync add Sonatype repository definition to your root pom.xml
+
+``` xml
+<repositories>
+.
+.
+ <repository>
+   <id>sonatype</id>
+   <name>Sonatype Groups</name>
+   <url>https://oss.sonatype.org/content/groups/public/</url>
+ </repository>
+.
+.
+</repositories>
+
+```
+
+Add Jest as a dependency to your project.
+
+
+``` xml
+<dependency>
+  <groupId>io.searchbox</groupId>
+  <artifactId>jest</artifactId>
+  <version>0.1.6</version>
```

Comment:
Should be 1.0.0-SNAPSHOT but well the whole thing has bigger problems than that.